sys/checksum/doc.txt

```diff
@@ -9,7 +9,7 @@
 
 /**
  * @defgroup    sys_checksum Checksum
- * @ingroup     sys
+ * @ingroup     sys_hashes
```

Comment:
Why do you put checksum in `sys_hashes`? I would guess that (as long as we keep both checksum/ and hashes/ in separate folders) we would keep both `@ingroup sys` ?

Reply:
As I have read, strictly speaking checksums are hashes. See for instance [this entry](https://en.wikipedia.org/wiki/List_of_hash_functions).
Although they are in different folder, I'm trying to organize them by topic :)

Reply:
I was referring to the mismatch between folder structure and documentation. But I agree that it looks way nicer this way. So the conclusion might be to consider file shifting which is not the purpose of this PR and maybe not wanted anyway.

Reply:
Ah, I see. Yes, could be an option. I would be in for moving the folder in another PR